torchvision/prototype/models/optical_flow/raft.py

```diff
@@ -46,6 +48,8 @@ class Raft_Large_Weights(WeightsEnum):
             "recipe": "https://github.com/pytorch/vision/tree/main/references/optical_flow",
             "sintel_train_cleanpass_epe": 1.3822,
             "sintel_train_finalpass_epe": 2.7161,
+            "kitti_train_per_image_epe": 4.5118,
+            "kitti_train_f1-all": 16.0679,
```

Comment:
Out of curiosity is this the standard F1 score?

Reply:
Not really, it's a kitti-specific metric, I saw it in different papers (and it's what they report in the leaderboards)

```py
"f1": ((epe > 3) & (relative_epe > 0.05)).float().mean().item() * 100,
```